doc/content/modules/user-manual/pages/release-notes/2025.4.0.adoc

```diff
@@ -43,6 +43,12 @@ action def ActionDef1 {
 - Fix an issue on import that set the default `TransitionFeatureKind` value on accept and do `TransitionFeatureMembership`
 Importing an "accept" `TransitionFeatureMembership` now correctly sets its kind to `trigger`.
 Importing a "do" `TransitionFeatureMembership` now explicitly sets its kind to `effect`.
+- Fix import of `TextualRepresentation` to clean _language_ and _body_ properties.
+They no longuer contains the extra characters :
```

Comment:
- contains => contain
Please consider installing a spell checker in your environment (e.g. Code Spell Checker in VSCode)

- characters : => characters:
Colons are not prefixed with a space in english.